Source/TestDafny/MultiBackendTest.cs

```diff
@@ -185,6 +181,10 @@ public class MultiBackendTest {
     }.Concat(options.OtherArgs);
 
     var (exitCode, outputString, error) = RunDafny(options.DafnyCliPath, dafnyArgs);
+    var compilationOutputPrior = new Regex("\r?\nDafny program verifier[^\r\n]*\r?\n").Match(outputString);
+    if (compilationOutputPrior.Success) {
+      outputString = outputString.Remove(0, compilationOutputPrior.Index + compilationOutputPrior.Length);
+    }
```

Comment:
This removes the "Dafny program verifier..." line from anywhere in the actual output, rather that prepending this line to the expected output (which had been done elsewhere in the code).

Reply:
Aha I see from the metatest change now how this is different, makes sense.

This does make me think we should really have an option to suppress that output rather than hacking around it here (i.e. https://github.com/dafny-lang/dafny/issues/3460), but that can be done later.